azure-pipelines/variables.yml

```diff
@@ -0,0 +1,16 @@
+variables:
+  windows_image: 'windows-2019'
+  linux_image: 'ubuntu-18.04'
+  mac_image: 'macOS-10.15'
+  CACHE_VCPKG_KEY: v4 | vcpkg-root
+  VCPKG_CMAKE: "$(Build.SourcesDirectory)/vcpkg/scripts/buildsystems/vcpkg.cmake"
+  #ARCH:
+  #TRIPLET:
+  #VCVARS:
+  SDL2_STATIC: ON
+  BUILD_TESTING: ON
+  BUILD_SHARED_LIBS: ON
+  BUILD_EXAMPLES: ON
+  BUiLD_SNAPSHOT: OFF
+  CMAKE_ARGS:  -DWITH_SDL2_STATIC=$(SDL2_STATIC) -DBUILD_TESTING=$(BUILD_TESTING) -DBUILD_SHARED_LIBS=$(BUILD_SHARED_LIBS) -DBUILD_EXAMPLES=$(BUILD_EXAMPLES) -DBUILD_SNAPSHOT=$(BUILD_SNAPSHOT)
+  CMAKE_IGNORE_PATH_WIN: "C:/MinGW/bin;C:/Strawberry/c/bin;C:/Program Files (x86)/LLVM/bin;C:/ProgramData/chocolatey/bin"
```

Comment:
based on this line on windows should be fairly possible to compile with GCC and CLang, quite easily....